page-officers.hbs

```diff
@@ -349,6 +349,14 @@ into the {body} of the default.hbs template --}}
                     <a target="_blank" class="icon-linkedin" href="https://www.linkedin.com/in/aezhang/"></a>
                     <a target="_blank" class="icon-github" href="https://github.com/aezhang"></a>
                   </div>
```

Comment:
Be careful to not forget the extra `</div>` here. Right it looks like you're adding your card as a part of Andrew's card.